### PR TITLE
[Fix] NOT the latest content release error message

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/22626

## Description
Added an error message for when the release notes check fails to fetch latest release tag, urging the user to merge from master.

## Screenshots
image

### CODE REVIEW

File: release_notes.py [hunk 1]
```diff
@@ -632,41 +632,45 @@ def main():
...
+                beta_res.append(beta_ans)
+
+        release_notes = "\n---\n".join(res)
+        beta_release_notes = "\n---\n".join(beta_res)
+        create_content_descriptor(args.version, args.asset_id, release_notes, args.github_token, beta_rn=beta_release_notes)
+
+        if missing_release_notes:
+            print_error("Error: some release notes are missing. See previous errors.")
+            sys.exit(1)
+    except RuntimeError:
```
Comment (anara123) [inline release_notes.py]: RuntimeError is too generic. We can miss other exceptions here.
which line exactly causes this exception/error?

File: release_notes.py [hunk 2]
```diff
@@ -632,41 +632,45 @@ def main():
...
+        print_error('Failed to get latest release tag. This may happen if your branch is not updated with '
```
Comment (anara123) [inline release_notes.py]: For external contributions we either should ignore this error or give them exact instructions how to pull from our origin master. 
This error doesn't contain such instructions.

File: release_notes.py (hunk 1 above)
Comment (DeanArbel) [reply release_notes.py]: @anara123 it shouldn't be too generic as it uses the exception that is being thrown in `run_command` function, which is called in line 636

File: release_notes.py (hunk 2 above)
Comment (DeanArbel) [reply release_notes.py]: I added the following instructions:
`Please run "git origin/master"`

**Comment by DeanArbel** [comment]
@kirbles19 would you mind giving in your 2 cents for the readability of the error message? (see the screenshot above)

**Pull request merged**